modules/geolocation/geolocation.py

```diff
@@ -129,3 +129,38 @@ def calculate_pixel_to_geo_mapping(self):
 
         # Return matrix product of mappedGeoMatrix and sourcePixelMatrixInverse
         return (mappedGeoMatrix.dot(sourcePixelMatrixInverse))
+
+    def map_location_from_pixel(self, pixels, coordinates):
+        """
+        Maps Geographical Location Coordinates in the destination image
+        
+        Parameters
+        -------
+            pixels : np.array(shape=(3,3))
+            coordinates : np.array(shape=(5,2))
+
+        Returns
+        -------
+            np.array(shape=(5,2))
+        """
+        dehomogenized = np.array([])
```

Comment:
Rename this to something better. Perhaps `geoCoordinates` given the input is currently `coordinates`?

Reply:
Renamed 'dehomogenized' to 'geoCoordinates'